src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PENamespaceSymbol.cs

```diff
@@ -190,6 +191,21 @@ protected void LoadAllMembers(IEnumerable<IGrouping<string, TypeDefinitionHandle
             LazyInitializeTypes(nestedTypes);
         }
 
+        private int GetQualifiedNameLength()
```

Comment:
Perhaps this method can be pushed to the NamespaceSymbol and made internal so that more code could take advantage of it in the future.